packages/ui/src/domains/LandingPageBuilder/LandingPageBuilderV2.tsx

```
@@ -0,0 +1,1098 @@
+	TabsContent,
+	TabsList,
+	TabsTrigger,
+} from "../../components/ui/tabs";
+import {
+	DragDropContext,
+	Droppable,
+	Draggable,
+	DroppableStateSnapshot,
+	DroppableProvided,
+	DropResult,
+} from "react-beautiful-dnd";
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

This import is only used as a type.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)